src/test/java/com/browserstack/examples/stepdefs/AbstractBaseSteps.java

```diff
@@ -5,11 +5,6 @@
 import com.browserstack.examples.tests.RunWebDriverCucumberTests;
 import com.browserstack.webdriver.core.WebDriverFactory;
 
-/**
- * Created with IntelliJ IDEA.
- *
- * @author Anirudha Khanna
- */
```

Comment:
Please edit the docs not remove them

Reply:
removed since we have not followed this throughout the repos... even for this repo ... it is not consistently followed across all files. can we descope for now?